vnext/proposals/HandleLightDarkTheme.md

```diff
@@ -0,0 +1,166 @@
+| 2 | Ship a separate Windows Native Module that enables a React Native developer to access our theme brushes |
+
+- The first feature enables a React Native developer to be conscious of the Windows theme change events, and alter their style(s) accordingly if desired
+- The second feature expands the theme customization further, by allowing an app-developer at the React Native layer to access our Windows-defined theme brushes in their styles
+
+## Basic examples
+
+### Example 1 : Set up and behaviors on theme changed
+In this example, we'll look at three things:
+
+- How to set up your React Native app to be style and event sensitive to the system themes
+- How to switch styles when a theme change has occurred
+- Handling a theme changed event
+
+#### Setting up your app to be sensitive to theme changes
+First import the Platform API into your React Native app.
+
+```JavaScript
+import { AppTheme } from 'react-native-windows'
+```
+
+Create a local variable to use in style conditionals or to reference elsewhere easily.
+
+```JavaScript
+class MyAppClass extends Component {
+  state = {
+    appThemeState: AppTheme.currentTheme,
+  };
+  ...
+}
+```
+
+#### Switching styles based on the app's theme
+If the app author wants to switch the style of their component manually based on the system's theme (Dark or Light), they can do so with CSS style conditionals.
+
+```JavaScript
+<TextInput
+    style={[styles.inputStyle, this.state.appThemeState == AppTheme.dark ? styles.darkInput : styles.lightInput]}
+</TextInput>
+```
+
+#### Handling a theme changed event
+In this case an app author would like to do something or preform some behavior when the app's theme has changed.
+
+> Note: AppThemeState will be a subclass of **NativeEventEmitter**.
+
+```JavaScript
+componentDidMount() {
+  Platform.currentTheme.addListener('themechanged', this.onAppThemeChanged);
+}
+
+onAppThemeChanged = (event) => {
+  /*Logic on theme changed goes here*/
+  this.setState({appThemeState: AppTheme.currentTheme});
+}
+```
+
+
+### Example 2 : Using Windows-defined theme brushes
+This example covers how to access and use the Windows sytem theme brushes and apply them in your CSS styles.
+
+#### Using theme brushes in a style
+
+```css
+const styles = StyleSheet.create({
+  title: {
+    fontSize: 19,
+    fontWeight: 'bold',
+    color: {windowsbrush: 'SystemControlPageTextBaseHighBrush'}
+  },
+});
+```
+Typing ``windowsbrush:`` will trigger the intellisense to scope to the system theme brushes available in native generic.xaml.
+
+#### Applying a system accent color variant
+In Windows, there are algorithmically generated accent colors - dubbed Light or Dark 1, 2, and 3. This example covers what it would look like to apply that using the windowsbrush object.
+
+```css
+const styles = StyleSheet.create({
+  title: {
+    fontSize: 19,
+    fontWeight: 'bold',
+    color: {windowsbrush: 'SystemAccentColorLight3'}
```

Comment:
if we're specifying the color, wouldn't it make more sense to expose the system brushes as color value consts in some object? e.g.

`color: WindowsBrushes.SystemAccentColorLight3`

Reply:
I like this, the consts can still be string values, but it supports intellisense better.